tests/test_fileread.py

```diff
@@ -0,0 +1,12 @@
+#adde the current working directory to the PATH
+import sys,os
+sys.path.append(os.getcwd())
+
+import pytest
+import os
+from ames_qsar.out_fileread import out_fileread
+
+def test_fileread():
+    #these are the test files that should be found
+    assert set(out_fileread(os.getcwd()+ '/tests')) == set(['coumarin_gas.out'\

```

Comment:
there is a correct way of getting package-local files for testing that we will look up